markdown/guide/deploying-engines.md

```diff
@@ -0,0 +1,22 @@
+## Deploying An Engine
+
+In many cases, the engine code will be served from a static server, typically a CDN.
+This server will probably not share the same root location as your application.
+In order to specify the engine's assets location, you can implement a `generateURI` function at the host application's build options file:
```

Comment:
The phrase "engine's assets location" reads a little odd to me, can we rephrase just as "engine asset location"?

Also, let's change "at the host application's build options file" to "in the host application's `ember-cli-build.js`"